scraper.rb

```diff
@@ -3,12 +3,28 @@
 require 'scraperwiki'
 require 'colorize'
 require 'combine_popolo_memberships'
+require 'require_all'
+require 'scraped_page_archive'
 
-require_relative 'lib/members.rb'
+require_rel 'lib'
 
 members_url = 'http://api.openhluttaw.org/en/memberships'
-members = Members.new(members_url).to_h[:members_of_the_lower_house]
 
-members.each do |member|
-  ScraperWiki.save_sqlite([:id], member.to_h)
+def all_members(url)
+  r = []
+  current_page = MembersRecords.new(
+    response: Scraped::Request.new(url: url).response
+  )
+  r << current_page.members_of_the_lower_house
+  while current_page.next
+    current_page = MembersRecords.new(
+      response: Scraped::Request.new(url: current_page.next).response
+    )
+    r << current_page.members_of_the_lower_house
+  end
+  r.flatten
+end
```

Comment:
This is all quite complex, but as it's not part of any public interface, I'm curious why you actually _need_ to build up the list of all members first, before writing them to the database, rather than simply iterating through all the pages and saving as you go.